Automattic/wp-calypso
# A4A: Implement Import WPCOM sites CTA functionality.

This PR implements the 'Import site' CTA button functionality. 

<img width="863" alt="Screenshot 2024-07-05 at 10 52 56 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/3617971c-61f2-4098-9ca4-92faa484d114">


Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/678

## Proposed Changes

* Update the useImportWPCOMSites hook to support multiple IDs. The hook will process sites in parallel.
* Update the Import WPCOM sites modal's CTA button to use the new importer.

## Testing Instructions

* Use the A4A live link below and go to `/sites` page.
* Click the 'Add site' dropdown menu and select 'Via WordPress.com` option.
* Select a few sites and click the 'import' button.
* Confirm that all selected sites are added to the dashboard.

## Pre-merge Checklist
